dts/arm/st/stm32f103Xe.dtsi

```diff
@@ -8,3 +8,26 @@
  */
 
 #include <st/stm32f103Xb.dtsi>
+
+/ {
+	soc {
+		spi2: spi@40003800 {
```

Comment:
This node could be moved inside stm32f103Xb.dtsi.

Reply:
done, but with a comment, that it needs to be removed in stm32f103tb.dtsi